exercises/csharp/TriviaGame/TriviaGame/Game.cs

```diff
@@ -40,71 +44,66 @@ public string CreateRockQuestion(int index)
 
         public bool IsPlayable()
         {
-            return (HowManyPlayers() >= 2);
+            return (HowManyPlayersInCurrentGame() >= 2);
         }
 
-        public bool Add(string playerName)
+        public bool AddNewPlayer(string playerName)
         {
-
-
-            players.Add(playerName);
-            places[HowManyPlayers()] = 0;
-            purses[HowManyPlayers()] = 0;
-            inPenaltyBox[HowManyPlayers()] = false;
+            listOfAllPlayers.Add(playerName);
+            playersPosition[HowManyPlayersInCurrentGame()] = 0;
+            playersPurses[HowManyPlayersInCurrentGame()] = 0;
+            inPenaltyBox[HowManyPlayersInCurrentGame()] = false;
 
             Console.WriteLine(playerName + " was added");
-            Console.WriteLine("They are player number " + players.Count);
+            Console.WriteLine("They are player number " + listOfAllPlayers.Count);
             return true;
         }
 
-        public int HowManyPlayers()
+        public int HowManyPlayersInCurrentGame()
         {
-            return players.Count;
+            return listOfAllPlayers.Count;
         }
 
-        public void Roll(int roll)
+        public void RollDice(int DiceRollResult)
         {
-            Console.WriteLine(players[currentPlayer] + " is the current player");
-            Console.WriteLine("They have rolled a " + roll);
+            Console.WriteLine(listOfAllPlayers[currentPlayer] + " is the current player");
+            Console.WriteLine("They have rolled a " + DiceRollResult);
 
             if (inPenaltyBox[currentPlayer])
             {
-                if (roll % 2 != 0)
+                if (DiceRollResult % 2 != 0)
```

Comment:
Improve this so '`% != 0`' is explained. It effectively means: IsThisNumberOdd doesn't it?